core/scripts/local/AdvanceEMPTime.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
nit: I think this should be in a separate PR.

Reply:
deleted